price_parser/parser.py

```diff
@@ -31,7 +31,6 @@ def fromstring(cls, price: Optional[str],
         Given price and currency text extracted from HTML elements, return
         ``Price`` instance, which provides a clean currency symbol and
         price amount as a Decimal number.
-
```

Comment:
What do you have against paragraphs? :stuck_out_tongue:

Reply:
That might be due to the auto-indentation of Pycharm. We can definitely undo it. 😄